digits/model/tasks/caffe_train.py

```diff
@@ -54,6 +54,14 @@ def upgrade_network(network):
         #TODO
         pass
 
+    @staticmethod
+    def set_mode(gpu):
+        if gpu>=0:
```

Comment:
Can we use `gpu=None` and `gpu is None` instead of `gpu=-1` and `gpu>=0`? Sorry for being nitpicky, but I feel like it's a little less cryptic and more pythonic.